### PR TITLE
Second practice

### PR DESCRIPTION
**Pull request by Berya1988**

### CODE REVIEW

File: src/main/java/org/example/App.java [hunk 1]
```diff
@@ -9,5 +9,6 @@
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
+        Test t = new Test ();
     }
```
Comment (berestoleg88) [inline src/main/java/org/example/App.java:13]: not bad:)

File: src/main/java/org/example/Test.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package org.example;
+
+public class Test {
+}
```
Comment (berestoleg88) [inline src/main/java/org/example/Test.java:4]: add something here :)

Comment (Berya1988) [reply src/main/java/org/example/Test.java:4]: it's fine

Comment (berestoleg88) [reply src/main/java/org/example/Test.java:4]: Ok

**Pull request merged**